toucan_connectors/common.py

```diff
@@ -122,7 +122,16 @@ def _handle_missing_params(elt, params):
     return query
 
 
-def render_raw_permissions(query, parameters):
+def apply_query_parameters(query, parameters):
+    """
+    Apply parameters to query
+
+    :param      query:       The query
+    :type       query:       { type_description }
+    :param      parameters:  The parameters
+    :type       parameters:  { type_description }
```

Comment:
I suggest explaining what the function does here instead of documenting the arguments.
I suggest:
```
Interpolate the query, which is a Jinja templates, with the provided parameters.
```

What we also can do to improve this doc is add type-hints to the signature:
 `def apply_query_parameters(query: str, parameters: dict): -> str`